### PR TITLE
[ci-app] Update Git Metadata Extraction

### PR DESCRIPTION
**Pull request by juan-fernandez**
### What does this PR do?
* Remove `git-repo-info`.
* Update tests.

### Motivation
* `git-repo-info` has been failing and is not reliable.

### CODE REVIEW

File: vendor/git-repo-info.js [hunk 1]
```diff
@@ -303,7 +306,14 @@ function getCommitData(gitPath, sha) {
   var objectPath = path.join(gitPath, 'objects', sha.slice(0, 2), sha.slice(2));
 
   if (zlib.inflateSync && fs.existsSync(objectPath)) {
-    var objectContents = zlib.inflateSync(fs.readFileSync(objectPath)).toString();
+    let objectContents = zlib.inflateSync(fs.readFileSync(objectPath)).toString();
+
+    // if there's a gpg signature, we remove it
```
Comment (rochdev) [inline vendor/git-repo-info.js:311]: Is this tested anywhere?

Comment (juan-fernandez) [reply vendor/git-repo-info.js:311]: no. We have no coverage at all for this file. Maybe I add coverage only for this bit? In any case, at this point I'm seriously considering removing this dependency altogether: it's just not reliable enough.

Comment (rochdev) [reply vendor/git-repo-info.js:311]: Yeah, I'd say if we're going to be significantly altering this dependency, and it's not deemed reliable anyway, let's just remove it and roll our own.

**Comment by juan-fernandez** [comment]
@rochdev so I went ahead and removed `git-repo-info` since it is not reliable. We'll need a way to access `.git` folder without `git` but that can be done later.

**Review by rochdev — APPROVED** [review]

**Pull request merged**